test/index.test.js

```diff
@@ -74,10 +71,10 @@ describe('DatePicker', () => {
       .create(<DatePicker style={style} value={new Date(DATE)} />)
       .toJSON();
 
-    expect(tree).toHaveProperty(
-      'props.style.backgroundColor',
-      style.backgroundColor,
-    );
+    expect(tree).toHaveProperty('props.style', [
```

Comment:
the `it` label of this test no longer makes sense because there is no `View` wrapper, could you fix that? Thanks!